### PR TITLE
Run gopherage xmlparsing for Coverage tests 

### PR DESCRIPTION
**Pull request by MushuEE**
Adding 
```
    <testcase class_name="go_coverage" name="OVERALL" time="0">
        <properties>
            <property name="coverage" value="57.9"></property>
        </properties>
```
Coverage property to junit results to be displayed in TestGrid. 

/hold There are two xml results being produced, I want only the one filtered and generated one.

### CODE REVIEW

File: config/jobs/kubernetes/sig-testing/coverage.yaml [hunk 1]
```diff
@@ -145,7 +149,11 @@ periodics:
         result=0
         ( cd hack/tools && GO111MODULE=on go install gotest.tools/gotestsum ) || result=$?
         make test KUBE_COVER=y KUBE_COVER_REPORT_DIR="${ARTIFACTS}" KUBE_TIMEOUT="--timeout=300s" || result=$?
-        grep -v -E 'zz_generated|generated\.pb\.go' "${ARTIFACTS}/combined-coverage.out" > "${ARTIFACTS}/filtered.cov" || result=$?
+        cd ../test-infra/gopherage
+        GO111MODULE=on go build .
+        export JUNIT=$(find ${ARTIFACTS} -type f -iname 'junit*.xml')
+        ./gopherage filter --exclude-path="zz_generated,generated\.pb\.go"  "${ARTIFACTS}/combined-coverage.out" > "${ARTIFACTS}/filtered.cov" || result=$?
+        ./gopherage junit --threshold 0 "${ARTIFACTS}/filtered.cov" > "${JUNIT}" || result=$?
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-testing/coverage.yaml:156]: If the goal is to ensure there are no junit.xml files other than the one we're generating, I would go about it this way.  As written if there were ever multiple junit.xml files I'm not sure what the `> ${JUNIT}` would do.
```suggestion
        find ${ARTIFACTS} -type f -iname 'junit*.xml' -exec rm {} \;
        ./gopherage filter --exclude-path="zz_generated,generated\.pb\.go"  "${ARTIFACTS}/combined-coverage.out" > "${ARTIFACTS}/filtered.cov" || result=$?
        ./gopherage junit --threshold 0 "${ARTIFACTS}/filtered.cov" > junit-coverage.xml || result=$?
```

Comment (MushuEE) [reply config/jobs/kubernetes/sig-testing/coverage.yaml:156]: There should only ever be one but I can make the changes to make it more robust 👍

**Pull request merged**